### PR TITLE
Update lane follower example

### PR DESCRIPTION
**Pull request by citlaligm**
## Overview
Updates the documentation for the lane follower example to reflect the change on the cpp client. This closes #249 

## Changes
Update how to read the map using the OpenDrive map and not the GeoJson and how to get the closest point

### CODE REVIEW

File: docs/cpp_client/cpp_examples.md [hunk 1]
```diff
@@ -167,12 +167,15 @@ state_config.undesired_tags = {""};
 sensors.push_back(std::make_shared<Sensor>(std::make_unique<StateConfig>(state_config)));
 ```
 
-These examples use the GeoJSON lanes from the monoDrive `Straightaway5k` map in 
-order to calculate the distance of the ego vehicle to the current lane. The 
-lanes can be read in as follows:
-
+These examples use the OpenDrive Map in order to calculate the distance of the ego vehicle to the current lane. The map can be read like this:
```
Comment (bradleynull) [inline docs/cpp_client/cpp_examples.md:170]:
```suggestion
These examples query the simulator for the OpenDrive Map definition, parse it using the client's map API, and query the resulting data structure to determine the target location for the ego vehicle. The map query to the simulator is sent and read here:
```

**Review by bradleynull — CHANGES_REQUESTED** [review]
@citlaligm Just a small change to the language of the example. Looks good other than that.

By the way, good catch on updating these! I thought we might let these slip through the cracks when we were first writing them :)

**Comment by citlaligm** [comment]
Brad, thanks for the review and the suggestion. Should be ready now.

**Review by bradleynull — APPROVED** [review]
@citlaligm Looks good now. Approved!

**Pull request merged**